### ISSUE TITLE
Project name is unauthorized use of a registered trademark?

### ISSUE THREAD

**Issue by lpar** [lotus/lotus#445]
Lotus® is [a registered trademark of IBM](http://www.ibm.com/legal/us/en/copytrade.shtml#section-L).

I'm pretty sure that your use is infringing, since web products under the Lotus brand have been a [core IBM product area](https://en.wikipedia.org/wiki/IBM_Lotus_Web_Content_Management).

Have you checked with IBM legal that the trademark is being abandoned?

**Comment by jodosha** [comment]
@lpar I haven't checked with them because this started as a side project and have nothing to do with IBM or their office suite. My intention is not to fool customers with the same name or take advantage of IBM popularity.

I named it after the spiritual meaning that this flower has for me. It's even tattooed on my arm.

---

I've noticed that you work for IBM on the Lotus Domino team. Are you speaking on behalf of your company?

**Comment by jxxcarlson** [comment]
jaam rek: Peace always

**Comment by lpar** [comment]
I don't work on the Lotus Domino team. I am not speaking on behalf of IBM. I was simply hoping to alert you to the problem so that you could resolve it without any need for lawyers being involved.

**Comment by jodosha** [comment]
@lpar Thank you for the clarification. I hope the same.
What's your suggestion?

**Comment by lpar** [comment]
Well, the clash is unfortunate and it will be a pain, but my suggestion would be to pick a new name. Maybe "nelumbo" after the genus of the sacred lotus flower? (_Nelumbo nucifera_)

Looks like there was an application to trademark [nelumbo](https://trademarks.justia.com/736/32/nelumbo-73632423.html) for laundry detergent, but it was abandoned, so you should be in the clear. (Disclaimer: This is not legal advice, as I'm not a lawyer either!)

(As an aside, the software company Lotus was [named after the lotus position](https://en.wikipedia.org/wiki/Lotus_Software#Origins), so it's also linked to the flower.)

**Comment by jxxcarlson** [comment]
If you do need to change, what about ‘lotus framework’ or some such composite?

**Comment by joneslee85** [comment]
I hate to think that we have to rename the project.

Marketing-wise, `nelumbo` is not very great as it sound awkward and hard to memorize.

I would not worry about this now until it has become a problem. Then we could think of other naming.

**Comment by jodosha** [comment]
@lpar "lawyers", "will be a pain".. The more I read what you write, the more this looks like an informal "cease and desist".

I can even rename the project _"nelumbo"_, which is horrible name btw, but there is one problem. I don't have money to register trademark, I don't have money to pay lawyers to defend that trademark. So if the next corporation wants to sue me, it can do it.

I'm exposing myself and my family with this project, by doing full time, unpaid work on Lotus, since July. I cannot even think what will happen if a letter from your lawyers will be delivered to my house.

I'm doing this "only" because I truly believe in Open Source, and giving back to the Community. I put all my efforts in it in the last three years, I never earned a dime from it, never tried to fool developers because we share the name.

My choice was unfortunate, but again, it's named after a spiritual meaning that it has for me.

This is the first time I feel discouraged and depressed about this project. Give to the Community and potentially get ruined as result. This industry is broken.

We'll evaluate what to do soon.

---

PS: the project will continue with the expected timeline, I don't know under which name in the future.

**Comment by Havvy** [comment]
http://www.bitlaw.com/trademark/common.html - If you use it first, you're protected.

This isn't really an existential threat - IBM doesn't want to destroy your project, they just don't want people confusing things as their project. The only company I've seen use trademark to try to kill others is King games.

**Comment by YorickPeterse** [comment]
It's worth mentioning that IBM owning the trademark of a name including the word "Lotus" does not per definition mean nobody else is allowed to use said word for a project. For example, the US (and many other countries) have a fair use policy that covers commonly used words, symbols and the likes. How this holds up in court is up to the lawyers and judges involved.

Without knowing what countries and laws are involved it's hard to get a definitive answer. I suggest contacting organizations such as the FSF, though I'm unsure if they help in these sort of cases. Also until IBM itself files a case I wouldn't worry about it too much, though that's just my personal view on the matter.

**Comment by jxxcarlson** [comment]
This is very good advice.

**Comment by YorickPeterse** [comment]
As an example, in the US alone there are already tons of trademarks containing the word "Lotus": http://tmsearch.uspto.gov/bin/showfield?f=toc&state=4806%3Azz5vri.1.1&p_search=searchss&p_L=50&BackReference=&p_plural=yes&p_s_PARA1=&p_tagrepl~%3A=PARA1%24LD&expr=PARA1+AND+PARA2&p_s_PARA2=Lotus&p_tagrepl~%3A=PARA2%24COMB&p_op_ALL=AND&a_default=search&a_search=Submit+Query&a_search=Submit+Query

In this list there are multiple instances of the literal word "Lotus" being trademarked by different entities.

The same applies to other commonly used words, e.g. "Ruby": http://tmsearch.uspto.gov/bin/showfield?f=toc&state=4806%3Azz5vri.1.1&p_search=searchss&p_L=50&BackReference=&p_plural=yes&p_s_PARA1=&p_tagrepl~%3A=PARA1%24LD&expr=PARA1+AND+PARA2&p_s_PARA2=Ruby&p_tagrepl~%3A=PARA2%24COMB&p_op_ALL=AND&a_default=search&a_search=Submit+Query&a_search=Submit+Query

**Comment by YorickPeterse** [comment]
Oh, apparently the links are bound to a session. Great. Either way, use the link http://tmsearch.uspto.gov/bin/gate.exe?f=login&p_lang=english&p_d=trmk, use "Basic Word Mark Search (New User) ", then search for "Lotus".

**Comment by nathanvda** [comment]
I am the author/maintainer of a rubygem called [cocoon](https://github.com/nathanvda/cocoon), and apparently there is a framework by Apache with the same name, and they have registered their trademarks, and they asked me to just place a very clear and explicit message that my code/gem was not related in any way to the Apache Cocoon project. Also: they reached out to me, as I was not aware of this at all.

Also I worked at a company where we guarded trademarks and helping searching for new trademark names, and @YorickPeterse is imho right. If a company would care about it's trademark, or would consider this project to be infringing, I would assume you would already be contacted. Personally I do not see how this project could be infringing for IBM. On the other hand, Disney considers _any_ use of any of their names infringing (for good reason probably). On the off chance they would contact you, just renaming to `Lotus.rb` would already put you in the clear.

> Note: while I worked at such a company, I am just a programmer, so this is just my opinion and has no legal basis whatsoever :smile:

**Comment by lpar** [comment]
Yes, you can have non-infringing use of the same trademark _in a different area of business_ where there won't be confusion. Look at Apple vs Apple — it was OK when Apple Computer made computers and Apple Corp was a record company, but once Apple Computer started selling computers that could make music, things got ugly.

So Lotus jewelry isn't going to clash with Lotus web software or Lotus Chinese restaurant. But that's why I mentioned IBM's products — the Lotus trademark has been specifically used by IBM for web software, including [web development frameworks](http://infolib.lotus.com/resources/oneui/2.1/docPublic/components/framework.htm). It confused _me_ for a moment when I first saw the name; that's how I ended up here, and the reason why I suspect it might lead to trouble.

You don't need to register a trademark, you don't need lawyers (yet), you don't need to spend money or get worked up. I'm just a bystander who happened to notice a likely trademark infringement, and to try and save you from unpleasantness I thought I'd alert you to the problem. It's quite possible I wasn't even supposed to do that, but I don't like seeing free software projects scared by legal letters.

Ultimately all I'm really saying is that personally, I think it might save you grief in the future if you come up with a name that hasn't been registered as a trademark and recently used for the exact same sort of project. I would have offered the same advice if your project was called WebLogic or WebObjects; the specifics of who owns what trademarks are just detail. You are, of course, free to ignore my personal advice. Or, you might just put a disclaimer in the README as @nathanvda suggests, and hope for the best.

As far as the lotus having deep personal meaning, I sympathize, and naming is hard. But, I can't help reflecting that a name is just a label, and that if you label a thing and cultivate strong attachment to that label... well, that's not really what the lotus flower is all about, is it?

**Comment by booch** [comment]
Don't worry about it until someone from IBM contacts you about it — and that doesn't seem very likely to happen. IBM is not using the Lotus trademark, and trademarks are unenforceable if they're not used. Of course, you don't have the time or money to fight them even if you're not doing anything wrong. But there's no sense in rolling over and playing dead if nobody is attacking you.

**Comment by smathy** [comment]
As others have mentioned, trademarks are registered in industries/categories, and the protection only exists within that industry that the owner continues to trade in.

That having been said, IBM definitely [hold the "Lotus" trademark](http://tmsearch.uspto.gov/bin/showfield?f=doc&state=4804:l7617i.9.7) in the relevant category ("Computer Programs and Related Documentation"), but here are the good things:

 1. The onus is on the trademark holder to enforce their trademark, there's no burden on you at all until they contact you.
 2. Even once that happens (if you wanted to fight it) they need to show that you're "engaged in commerce" in the jurisdiction they hold the trademark in, this is very clear for a non-FOSS project/product/company but less so for FOSS.
 3. They also need to show that there exists a likelihood of confusion, this is often very hard to show for non-competitors. This point also includes the requirement that they're actively using the trademark, if they're not using it then there can't be any confusion.
 4. That brings me to the biggest thing in your favor, IBM are pulling back from their Lotus trademark and name. Domino is now [IBM Domino](http://www-03.ibm.com/software/products/en/ibmdomino) and even Notes is [IBM Notes](http://www-03.ibm.com/software/products/en/ibmnotes) now. Their existing usage of "Lotus" is mostly as an umbrella brand for a whole suite of software ([protected by **this** trademark btw](http://tmsearch.uspto.gov/bin/showfield?f=doc&state=4804:l7617i.9.8)) and you're not in breach of that at all.

So, my summary, wait until they contact you anyway, and even then if you wanted I think you have grounds to force the issue.

**Comment by smathy** [comment]
Also, delete this issue so they don't find it when they google for IBM/Lotus/trademark ;)

**Comment by nathanvda** [comment]
@booch and @smathy :+1:

**Comment by jdickey** [comment]
Ummm…IANLAL, but I'd think that deleting or otherwise obscuring this discussion after the fact, in light of the Wayback Machine and other technology, *might* lead an ambitious young lawyer in the Stygian depths of Big Blue to try to score points by saying "hey! they knew they were potentially infringing; whether or not they actually were, *they tried to cover it up!* Let's bash their teeth in!" Shit like that has happened with far more tenuous cause.

**tl;dr** As in music, when you make a mistake, make it *loudly*.

**Comment by solyaris** [comment]
Ciao Luca!
first of all: your work on Lotus is great and i keep t this opccasion to thank you!

back to the "issue", I see your points, and yes, I understand the love for word 'lotus' because spiritual meanings, 

but let consider this remote legal trap in a positive way: my proposal could be to just contact IBM in some way, trying a no-cost agreement with them. As someone here state, maybe just a clarification note could be sufficient. 

but i want to exaggerate: this contact with IBM could be the occasion to ask them a sponsorhip for Lotus! Why not! IBM is in some way (I hope so) interested in Ruby realms (see rumorsa about possible contacts between Yukihiro Matsumoto and IBM). 
I believe Lotus is a splendid project an it could deserve attention by IBM :)
that could be invest more in software innovation and less in cloud computing marketing (my personal opinion) :)

respect
giorgio

**Comment by bradland** [comment]
IANAL, but I have been on both sides of trademark litigation. Smathy's summary of points are good, but I disagree with his advice to delete this issue. Jdickey is correct that this material would ultimately be considered material and subject to discovery (which is where the participants in litigation have to share information) in any potential litigation. The only way it would not be subject to discovery is if you had asked an attorney, rather than the internet.

Which leads me to another piece of advice. Soliciting legal advice in a public forum is a really bad idea. You're putting yourself in a position to discuss legal matters in a medium that is afforded zero protection from exploration by a potential legal opponent. I realize that this is antithetical to the open source mindset, but if you want to avoid trouble, you have to be prepared to switch contexts when working on your software versus running your organization.

My final advice would be this: don't obsess over future risk. Much of what has been said here — with regard to trademarks and their enforceability based on category/industry — is correct. However, the only action that really matters is that of the mark holder. lpar's comments are just that: comments. Just like some people can't help but say something when they see someone litter, lpar felt compelled to let you know that you're treading on questionable ground with regard to IBM's trademark. It's good information, but it carries no weight, legally speaking.

Until IBM comes knocking, or sells the trademark to someone else who cares, you don't have an issue. The counterpoint to this would be that you need to consider this question: will renaming the project get harder or easier with time?

If you are certain it will get harder over time, then rename the project now. Your chances of defending your use of the Lotus name against IBM are not good; unless you can find some outside source of funding to fight the case. That seems unlikely given that the Lotus trademark is in an industry so similar to your use case. The FSF or EFF are unlikely to come to your aid if they feel the chances of success are low, or that you should have been able to anticipate the issue.

If you feel that it makes no difference (relative to the difficulty of said task) whether you change the name now or at some future date, then keep on truckin'. At the time a mark holder comes knocking, it is extremely unlikely that they will take any action beyond issuing a cease & desist order. At that point, you can decide to change the name or fight it. If you were a commercial entity with substantial assets, they might seek to reclaim some of the benefit you had received from the abuse of their mark, but given that this is an open source project, it is unlikely that they'll waste the man hours with anything more than a boilerplate letter.

**Comment by jodosha** [comment]
Thank you all for the suggestions. I'm in contact with people that can give me a legal advice.

**Comment by jodosha** [comment]
We're changing name from Lotus to Hanami.

https://twitter.com/hanamirb/status/689861441361448960
